### PR TITLE
Output only main

### PR DESCRIPTION
**Pull request by lcawl**
This PR stops generating "master" documentation, so that we can find all the broken links.

### CODE REVIEW

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by szabosteve** [comment]
@elasticmachine run elasticsearch-ci/docs

File: conf.yaml [hunk 1]
```diff
@@ -554,10 +571,11 @@ contents:
                   -
                     repo:   docs
                     path:   shared/attributes.asciidoc
+                    map_branches: *mapMasterToMain
               - title:      Perl Client
                 prefix:     perl-api
                 current:    master
-                branches:   [ master, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
+                branches:   [ {master: main}, 8.6, 8.6, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```
Comment (szabosteve) [inline conf.yaml:578]:
```suggestion
                branches:   [ {master: main}, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```

File: conf.yaml [hunk 2]
```diff
@@ -596,7 +615,7 @@ contents:
               - title:      Rust Client
                 prefix:     rust-api
                 current:    master
-                branches:   [ master, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
+                branches:   [ {master: main}, 8.6, 8.6, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```
Comment (szabosteve) [inline conf.yaml:618]:
```suggestion
                branches:   [ {master: main}, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0 ]
```

File: conf.yaml [hunk 3]
```diff
@@ -1055,12 +1088,13 @@ contents:
...
                     path:   shared/versions/stack/{version}.asciidoc
+                    map_branches: *mapMasterToMain
               - title:      Enterprise Search Node.js client
                 prefix:     enterprise-search-node
                 current:    *stackcurrent
-                branches:   [ {main: master}, 8.6]
+                branches:   [ main, 8.6]
                 live:       [ main, 8.6 ]
-                index:      packages/enterprise-search/docs/index.asciidoc
+                index:      packages/enterprise-search-node/index.asciidoc
```
Comment (szabosteve) [inline conf.yaml:1097]:
```suggestion
                index:      packages/enterprise-search/index.asciidoc
```

**Comment by szabosteve** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by szabosteve** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by szabosteve** [comment]
@elasticmachine run elasticsearch-ci/docs

File: conf.yaml [hunk 4]
```diff
@@ -605,6 +624,7 @@ contents:
                   -
                     repo:   elasticsearch-rs
                     path:   docs/
+                    map_branches: *mapMainToMaster
```
Comment (szabosteve) [inline conf.yaml:627]:
```suggestion
```

**Comment by szabosteve** [comment]
@elasticmachine run elasticsearch-ci/docs

File: conf.yaml [hunk 5]
```diff
@@ -236,17 +239,20 @@ contents:
...
                 repo:   docs
                 path:   shared/settings.asciidoc
+                map_branches: *mapMasterToMain
           - title:      Elastic Common Schema (ECS) Reference
             prefix:     en/ecs
             current:    8.7
-            branches:   [  {main: master}, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
-            live:       [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
+            branches:   [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
+            live:       @stacklivemain
```
Comment (lcawl) [inline conf.yaml:255]:
```suggestion
            live:       *stacklivemain
```

File: conf.yaml [hunk 6]
```diff
@@ -236,17 +239,20 @@ contents:
...
                 repo:   docs
                 path:   shared/settings.asciidoc
+                map_branches: *mapMasterToMain
           - title:      Elastic Common Schema (ECS) Reference
             prefix:     en/ecs
             current:    8.7
-            branches:   [  {main: master}, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
-            live:       [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
+            branches:   [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
+            live:       *stacklivemain
```
Comment (lcawl) [inline conf.yaml:255]:
```suggestion
            live:        [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
```

File: conf.yaml [hunk 7]
```diff
@@ -236,17 +239,20 @@ contents:
...
                 repo:   docs
                 path:   shared/settings.asciidoc
+                map_branches: *mapMasterToMain
           - title:      Elastic Common Schema (ECS) Reference
             prefix:     en/ecs
             current:    8.7
-            branches:   [  {main: master}, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
-            live:       [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
+            branches:   [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
+            live:        [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
```
Comment (lcawl) [inline conf.yaml:255]:
```suggestion
            live:       [ main, 8.7, 8.6, 8.5, 8.4, 8.3, 1.12 ]
```

**Comment by kilfoyle** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild --warnlinkcheck

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild warnlinkcheck

**Pull request closed without merging**